### ISSUE TITLE
As an SU Admin accepting an application, I would like to know that I have hit a character limit before completing my notification message so that I don't have to do it twice.

### ISSUE THREAD

**Issue by ASprinkle** [USDAForestService/fs-open-forest-platform#1143]
## Notes

Currently an administrator is allowed to type a very long message to notify the applicant.  Upon completion of typing the message, the admin is greeted with an error message (screen shot) that tells them to use less than 255 characters.  At that point, the admin has to edit their message and try again.

## Acceptance Criteria
- [ ] Notification field is not larger than the character limit.
- [ ] The admin knows when they've reached their character limit due to validation expression (red border, and error notification text).

## Tasks
- [ ] needs DEV refinement.
- [ ] PO approved

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by ASprinkle** [comment]
What are we defining the character limit as?

**Comment by ASprinkle** [comment]
@Dmac26  @briandavidson  I am putting a "high priority" label on this story.  I learned today that one of the  MBS special use administrators attempted to put our first application on HOLD status and notify the applicant, but they were not able to do so because they received an error.  The error occured to them after they typed their hold message, and selected "hold and notify applicant".  So admins are using (or attempting to use) these functions.

**Comment by ASprinkle** [comment]
@Dmac26 @briandavidson can one of you verify that the Pull requests meet technical definition of done and check the appropriate box for this story above? Also, in the pull request it looks like there are some unchecked boxes in the story but I am not sure if I was looking at the right thing.

**Comment by briandavidson** [comment]
@ASprinkle I verified everything was complete in the PR, boxes are checked, should be g2g!

**Comment by mtlaney** [comment]
blocked by #1196
